lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerImplementation.java

```diff
@@ -52,60 +52,61 @@
-            @Parameter(RemoteConfiguration.OrderingGuarantee.class) final boolean orderingGuarantee,
-            @Parameter(RemoteConfiguration.NumberOfTries.class) final int numberOfTries,
-            @Parameter(RemoteConfiguration.RetryTimeout.class) final int retryTimeout,
-            final LocalAddressProvider localAddressProvider,
-            final TransportFactory tpFactory,
-            final TcpPortProvider tcpPortProvider) {
+        @Parameter(RemoteConfiguration.ManagerName.class) final String name,
+        @Parameter(RemoteConfiguration.HostAddress.class) final String hostAddress,
+        @Parameter(RemoteConfiguration.Port.class) final int listeningPort,
+        @Parameter(RemoteConfiguration.MessageCodec.class) final Codec<T> codec,
+        @Parameter(RemoteConfiguration.ErrorHandler.class) final EventHandler<Throwable> errorHandler,
+        @Parameter(RemoteConfiguration.OrderingGuarantee.class) final boolean orderingGuarantee,
+        @Parameter(RemoteConfiguration.NumberOfTries.class) final int numberOfTries,
+        @Parameter(RemoteConfiguration.RetryTimeout.class) final int retryTimeout,
+        final LocalAddressProvider localAddressProvider,
+        final TransportFactory tpFactory,
+        final TcpPortProvider tcpPortProvider) {
 
     this.name = name;
     this.handlerContainer = new HandlerContainer<>(name, codec);
 
     this.reRecvStage = orderingGuarantee ?
-                new OrderedRemoteReceiverStage(this.handlerContainer, errorHandler) :
-                new RemoteReceiverStage(this.handlerContainer, errorHandler, 10);
+        new OrderedRemoteReceiverStage(this.handlerContainer, errorHandler) :
+        new RemoteReceiverStage(this.handlerContainer, errorHandler, 10);
 
-    this.transport = tpFactory.newInstance(
-                hostAddress, listeningPort, this.reRecvStage, this.reRecvStage, numberOfTries, retryTimeout,
-                tcpPortProvider);
+    this.transport = tpFactory.newInstance(hostAddress, listeningPort,
+        this.reRecvStage, this.reRecvStage, numberOfTries, retryTimeout, tcpPortProvider);
 
     this.handlerContainer.setTransport(this.transport);
 
-    this.myIdentifier = new SocketRemoteIdentifier(
-                (InetSocketAddress) this.transport.getLocalAddress());
+    this.myIdentifier = new SocketRemoteIdentifier((InetSocketAddress)this.transport.getLocalAddress());
 
     this.reSendStage = new RemoteSenderStage(codec, this.transport, 10);
 
     StageManager.instance().register(this);
-    LOG.log(Level.FINEST, "RemoteManager {0} instantiated id {1} counter {2} listening on {3}:{4}. " +
-                "Binding address provided by {5}",
-                new Object[]{this.name, this.myIdentifier, COUNTER.incrementAndGet(),
-                        this.transport.getLocalAddress().toString(),
-                        this.transport.getListeningPort(), localAddressProvider}
```

Comment:
The port is already being printed as part of the local address. Original log message looked like `host:port:port` i.e. the port was printed twice